### DISCUSSION TITLE
Index on ToDo vs Notes or Both

### DISCUSSION THREAD

**Discussion by Atsiekratsie** [kudrykv/latex-yearly-planner#73] [General]
Hi, great work here! Really loving the work and my Supernote is loving it :-) 

Though in the examples there is an index on to do's. When I download the prepared files in the discussion, its all about notes. Would someone be able to run a version that looks like the example with the to do's? Or perhaps even both? I got everything running but run into build problems (error during build process). Thanks in advance!

**Comment by kudrykv** [comment]
Hi @Atsiekratsie!

Yeah, the pictures are outdated.

I'll search for the commit that has those notes and re-run the planner in it with the current year.

**Reply by Atsiekratsie** [reply]
any luck? Thanks in advance!

**Reply by Atsiekratsie** [reply]
I hate to bother you, but did you manage to re-run?

**Reply by kudrykv** [reply]
holy hell, I'm sorry for such a long wait 😞
[v0.0.7](https://github.com/kudrykv/latex-yearly-planner/tree/v0.0.7) has to-dos and notes.
Do you need help compiling it?

**Reply by alextien** [reply]
That's cool. Could you help to build the example pdf with both todo and note tap for those they don't know how to build like me.

Thank you very much.

**Reply by Atsiekratsie** [reply]
Yes please! Thanks again!

**Reply by kudrykv** [reply]
[2023.sn_a5x.planner.daily-with-cal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/12578000/2023.sn_a5x.planner.daily-with-cal.pdf)
[2024.sn_a5x.planner.daily-with-cal.pdf](https://github.com/kudrykv/latex-yearly-planner/files/12578001/2024.sn_a5x.planner.daily-with-cal.pdf)

**Reply by kudrykv** [reply]
To build it (assuming go and latex are in the system):
1. clone
2. checkout to `v0.0.7` tag
3. comment out redundant configurations in the `release.sh`. Then, run `./release.sh`
